### ISSUE TITLE
Double check on dynamic layout for localization in the preferences

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#312]

**Comment by zepumph** [comment]
Double check on the grab drag interaction string.

**Comment by arouinfar** [comment]
@zepumph it looks like `model.strings.sceneryPhet.key.toGrabOrReleaseStringProperty` is associated with this interaction hint. Editing this property in Studio does not actually change the string in the sim, though.
![image](https://user-images.githubusercontent.com/8419308/194625599-528d68e2-8d45-4778-9894-186e0727db4a.png)

I spent some time customizing the preferences strings. Generally everything looks good, but I did notice a few things:

* `strings.joist.preferences.tabs.audio.sounds.extraSounds.descriptionStringProperty` doesn't appear to have a `maxWidth`. I haven't created a realistic layout problem, so this may be just fine.
![image](https://user-images.githubusercontent.com/8419308/194626522-d0810865-44fb-4b95-8132-66b72669f82d.png)
* When `strings.joist.preferences.tabs.audio.voicing.descriptionStringProperty` becomes long, the toggle can get too close to the Sounds controls.
![image](https://user-images.githubusercontent.com/8419308/194627400-79aaf41d-aba8-4ea6-b4f0-b656976ab943.png)
* None of the controls under the Voicing toggle (Sim Voicing Options, Toolbar, etc.) have instrumented strings, but I think that's because the strings don't appear in the translation utility. Since Voicing is English-Only, that seems okay to me.
* We eliminated the General tab in favor of the Overview and Simulation tabs, so  `model.strings.joist.preferences.tabs.general` shouldn't exist, and its current contents need to get moved over to the Overview section.

**Comment by zepumph** [comment]
> * `strings.joist.preferences.tabs.audio.sounds.extraSounds.descriptionStringProperty` doesn't appear to have a `maxWidth`. I haven't created a realistic layout problem, so this may be just fine.

Right! Here is how it looks with a max height of 100 (lines wrap at 300):
![image](https://user-images.githubusercontent.com/6856943/194651199-e56324df-dc89-4cc2-94f3-52482d045639.png)

![image](https://user-images.githubusercontent.com/6856943/194651212-f14de6a3-4ebc-485e-8520-8369ce91dc9b.png)

> * When `strings.joist.preferences.tabs.audio.voicing.descriptionStringProperty` becomes long, the toggle can get too close to the Sounds controls.

I added spacing to the HBox: 
![image](https://user-images.githubusercontent.com/6856943/194653994-0c8827de-6141-417b-b4e3-a5e6bacfbef7.png)

>None of the controls under the Voicing toggle (Sim Voicing Options, Toolbar, etc.) have instrumented strings, but I think that's because the strings don't appear in the translation utility. Since Voicing is English-Only, that seems okay to me. 

This was a purposeful decision that I was originally confused by over in https://github.com/phetsims/joist/issues/852. Yes we don't want to cause bloat in i18n when it isn't actually supported.



> * We eliminated the General tab in favor of the Overview and Simulation tabs, so  `model.strings.joist.preferences.tabs.general` shouldn't exist, and its current contents need to get moved over to the Overview section.

Yes, but unfortunately it is very very dangerous to rename/change string key names. Since these strings are in published sims, we are a bit stuck. Sorry about that!

**Comment by arouinfar** [comment]
> Right! Here is how it looks with a max height of 100 (lines wrap at 300):
> I added spacing to the HBox:

Those both look good in master, thanks!

> Yes, but unfortunately it is very very dangerous to rename/change string key names. Since these strings are in published sims, we are a bit stuck. Sorry about that!

Good to know! Thanks for the clarification, I'll keep that in mind.

Anything else to do here?
